pkg/generate/app/cmd/newapp.go

```diff
@@ -561,7 +552,17 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
 
 func (c *AppConfig) validate() (cmdutil.Environment, cmdutil.Environment, error) {
 	var errs []error
-
+	// Environment File where the parameters are stored
+	if len(c.EnvironmentFile) != 0 {
+		varList, err := app.GetEnvironmentVars(c.EnvironmentFile)
+		if (err != nil){
+			return nil, nil, fmt.Errorf("Error reading file %v-%v", c.EnvironmentFile, err)
```

Comment:
Error output looks a bit cluttered with that format. It looks something like this:
error: Error reading file sample:open sample: no such file or directory. 

So I thought of changing it to:
error: Error reading file sample-open sample: no such file or directory.